docs/features/selective_consolidation_plan.md

```
@@ -0,0 +1,156 @@
+    *   [x] Display "Subdomains Parsed" (count of URLs in `jobStatus.urls`).
+        > Changes: `components/JobStatsSummary.tsx`, Line [25], [41-46]
+    *   [x] Display "Pages Crawled" (count of URLs with status `completed` in `jobStatus.urls` *after* crawl).
+        > Changes: `components/JobStatsSummary.tsx`, Line [26], [49-55]
+    *   [x] Display "Data Extracted" (fetch size from metadata or calculate from `.md` file).
+        > Changes: `components/JobStatsSummary.tsx`, Line [31], [58-64] (Placeholder 'N/A' used)
+    *   [x] Display "Errors Encountered" (count of URLs with `discovery_error` or `crawl_error` status).
+        > Changes: `components/JobStatsSummary.tsx`, Line [27], [67-73]
+    *   [x] Integrate this component into `app/page.tsx`.
+        > Changes: `app/page.tsx`, Line [6], [347] (Replaced `ProcessingBlock`)
+
+---
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)